modules/integration/tests-common/admin-clients/src/main/java/org/wso2/mb/integration/common/clients/operations/utils/AndesClientUtils.java

```diff
@@ -316,24 +316,18 @@ public static void createMockFile(String filePathToRead, String filePathToCreate
             }
 
         }
+
+        //If already exists, deleting the file
+        if (deleteRandomFile(filePathToCreate)) {
+            log.info("File requested to create already exists. Deleted file: " + filePathToCreate);
```

Comment:
what happens if file was not deleted for some reason... I think we should fail the test case...

AssertTrue(deleteRandomFile(filePathToCreate), "File deletion failed")